federation/backends/minimal_profile.md

```diff
@@ -0,0 +1,194 @@
+# Minimal profile for openEO backends
+This document describes the requirements for a backend that aims to provide minimal functionality for access and processing of gridded
+earth observation data.
+
+This document does not aim to contradict the openEO API, and when this is perceived to be the case, the definitions in the openEO API get precedence.
+It is possible that requirements in this document do define additional constraints on top of the openEO API.
+
+This profile is established because the openEO web service API is quite liberal if interpreted strictly. So a web service can be compliant to the API 
+specification, but still be mostly incompatible with other backends, and existing usage patterns. By following this profile, implementors have a better
+view on what is important.
+
+## Scope
+This minimal profile aims to help initial openEO implementations, that seek a least effort approach to achieve minimal compatibility with other backends.
+Requirements in this profile are not sufficient for backends in the openEO platform federation.
+
+## Mandatory endpoints
+
+In the openEO API, all endpoints are optional and discoverable. These are the endpoints that are required by this profile:
```

Comment:
Don't you also need `GET /` and (not sure) `GET /.well-known/openeo`? It's not listed in the endpoint list, but should likely be mentioned, right?

Reply:
indeed, added it.